staging/src/k8s.io/kubectl/pkg/cmd/version/version.go

```diff
@@ -74,6 +74,10 @@ func NewCmdVersion(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *co
 		Long:    i18n.T("Print the client and server version information for the current context."),
 		Example: versionExample,
 		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) > 0 {
```

Comment:
Maybe it would be better to wrap this check in a function like in here;

https://github.com/kubernetes/kubernetes/blob/9b09612d1b0a6ec441833f928cbb205d936da59c/staging/src/k8s.io/kubectl/pkg/cmd/diff/diff.go#L123

Reply:
I think so. @jlsong01

Reply:
fixed